filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,54 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
+  license: CC-0
+doc-ref: https://github.com/AgentD/squashfs-tools-ng/blob/master/doc/format.txt
+seq:
+  - id: superblock
+    type: superblock
+types:
+  superblock:
+    seq:
+      - id: magic
+        contents: 'hsqs'
+      - id: inode_count
+        type: u4
+      - id: mod_time
+        type: u4
+      - id: block_size
+        type: u4
+      - id: frag_count
+        type: u4
+      - id: compressor
+        type: u2
+      - id: block_log
+        type: u2
+      - id: flags
+        type: u2
+      - id: id_count
+        type: u2
+      - id: version_major
```

Comment:
BTW it may make sense to create a `version` struct.

Reply:
If there were other versions, but there aren't.

Reply:
Even now, because it will allow to put them in the same field called `version`. It is easier to read.

Reply:
I prefer the .ksy file to closely follow the spec with no surprises.

Reply:
I don't think it is a good approach. IMHO we must always improve if we can.